plugins/autostandby.koplugin/main.lua

```diff
@@ -114,6 +114,8 @@ function AutoStandby:onInputEvent()
     UIManager:scheduleIn(AutoStandby.delay, AutoStandby.allow, AutoStandby)
 end
 
+AutoStandby.onPageUpdate = AutoStandby.onInputEvent
```

Comment:
No, this is a PocketBook-only plugin that does a lot of weird crap, we don't want to rock the boat, it's already messy enough there.